README.md

```diff
@@ -65,6 +65,11 @@ No *inbound* access to the agent is required.
 See [Getting started with HTTPS proxies](https://github.com/adastradev/data-ingestion-agent/blob/master/docs/HttpsProxy.md) for more information.
 
 ### Query Preview
+Note: To see required table/field access for each integration type, see the following documentation:
+- [Banner](https://github.com/adastradev/data-ingestion-agent/blob/master/docs/DataAccess/Banner.md)
+- [DegreeWorks](https://github.com/adastradev/data-ingestion-agent/blob/master/docs/DataAccess/DegreeWorks.md)
+- [PeopleSoft](https://github.com/adastradev/data-ingestion-agent/blob/master/docs/DataAccess/PeopleSoft.md)
+
```

Comment:
For future awesomeness it'd be cool to have this section auto-updated as new integrations come on board (specifically if 3rd parties start extending it). For now this is awesome though.

Reply:
Agree, though it seems fine for now. We might separate them soon if we add use cases for Astra Schedule as well (as opposed to Platinum data sets)